src/marketplace.ts

```diff
@@ -26,6 +17,24 @@ export interface MarketplaceClientParams {
   wallet: Wallet
 }
 
+export type EditTokensParams = MarketplaceBaseParams & {
+  originalAuctionHouses: AuctionHouse[]
+  tokens: { address: string }[]
+}
+
+interface MarketplaceBaseParams {
+  logo: { uri: string; type?: string; name?: string }
+  banner: { uri: string; type?: string; name?: string }
+  subdomain: string
+  name: string
+  description: string
+  transactionFee: number
+  creators: { address: string }[]
+}
+
+export type EditMarketplaceParams = MarketplaceBaseParams & {
+  auctionHouses: { address: string }[]
+}
```

Comment:
Lets make the auctionHouses field option instead of a separate edit type.

Reply:
Ok